server/constants/constants.go

```diff
@@ -95,6 +95,10 @@ const (
 	GetTasks    = "/%s/_apis/wit/workitems"
 	StaticFiles = "%s/plugins/%s/static/%s"
 
+	// Query params sent to Azure DevOps APIs
```

Comment:
```suggestion
	// Query params sent from Azure DevOps APIs
```

Reply:
shouldn't it be this as we are receiving requests?

Reply:
No, we are sending these to Azure